mf_holdings.json

```diff
@@ -2,50 +2,37 @@
   "status": "success",
   "data": [
     {
-      "folio": "123123/123",
-      "fund": "Kotak Select Focus Fund - Direct Plan",
-      "tradingsymbol": "INF174K01LS2",
-      "average_price": 30.729,
-      "last_price": 33.014,
-      "pnl": 594.769,
-      "last_price_date": "2016-11-11",
-      "quantity": 260.337
+      "folio": "3108290884",
```

Comment:
What is changing in this file?

Reply:
Figured it out, I think its from #22, can you rebase this branch with `main` once?